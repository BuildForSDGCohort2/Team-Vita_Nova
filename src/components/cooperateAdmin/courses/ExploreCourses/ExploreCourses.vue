<template>
  <v-responsive>
    <v-container style="max-width: 1000px" grid-list-md>
      <div :style="{'font-family': 'IBM Plex Sans'}">
        <v-row>
          <v-col class="d-flex" cols="12" sm="2" md="2" >
            <v-select
                v-model="select"
                @change="selectChange"
                class="select my-2"
                :items="dropdown_font"
                label="Sort By:"
                flat
                outlined
                dense
            ></v-select>
          </v-col>
          <v-col cols="12" md="5">
          </v-col>
          <v-col cols="12" md="5" >
            <v-text-field
                append-icon="mdi-magnify"
                label="Search by Course name"
                solo-inverted
                flat
                :style="{'alignment': 'right'}"
                v-model="searching"
            ></v-text-field>
          </v-col>
        </v-row>
        <v-row>
          <v-col cols="12" md="6">
            <h3 :style="{'text-transform': 'uppercase'}">Assigned Courses</h3>
          </v-col>
        </v-row>
        <v-row>
          <v-col cols="12" md="6" v-for="(card, index) in cards.assignedCourses" :key="index">
            <div>
              <CourseCard :cards="card"/>
            </div>
          </v-col>
        </v-row>
        <v-row class="all-courses">
          <v-col cols="12">
            <h3 class="all-courses-header" :style="{'text-transform': 'uppercase'}">Existing Courses</h3>
          </v-col>
        </v-row>
        <v-row>
          <v-col cols="12" md="6" v-for="(card, index) in cards.existingCourses" :key="index">
            <div>
              <CourseCard :cards="card" :index="index"/>
            </div>
          </v-col>
        </v-row>
      </div>
    </v-container>
  </v-responsive>
</template>

<script>
  import CourseCard from "../CourseCard";
  export default {
    name: "ExploreCourses",
    data() {
      return {
        dropdown_font: ['Arial', 'Calibri', 'Courier', 'Verdana'],
        searching: '',
        select: ''
      }
    },
    methods: {
      selectChange() {
        console.log('inside selectChange')
      }
      // viewAll(type) {
      //   if (type === 'myCourses'){
      //     console.log(this.cards.assignedCourses);
      //   }
      //   else {
      //     console.log(this.cards.existingCourses);
      //   }
      // },
      // search(){
      //   const searchCourses = [];
      //   if(this.searching){
      //     for (const i in this.cards.assignedCourses) {
      //       if (this.cards.assignedCourses.hasOwnProperty(i) && this.cards.assignedCourses[i].course.toLowerCase().match(this.searching.toLowerCase())) {
      //         searchCourses.unshift(this.cards.assignedCourses[i].course);
      //       }
      //     }
      //   }
      //   console.log(searchCourses);
      // }
    },
    components: {
      CourseCard
    },
    computed: {
      cards() {
        return {
          assignedCourses: [
            {
              course: "Introduction to Robotics",
              subtitle: "21 Modules Included",
              company: "Hobo Tech",
              courseImageLink: 'https://images.unsplash.com/photo-1454165804606-c3d57bc86b40?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60',
              rank: 4.8
            }
          ],
          existingCourses: [
            {
              course: "Introduction to Python Programming",
              subtitle: "21 Modules Included",
              courseImageLink: 'https://images.unsplash.com/photo-1481627834876-b7833e8f5570?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60',
              rank: 4.8
            },
            {
              course: "The Evolution of Design",
              subtitle: "21 Modules Included",
              courseImageLink: 'https://images.unsplash.com/photo-1471107340929-a87cd0f5b5f3?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=666&q=80',
              rank: 4.8
            },
            {
              course: "Art & Design Principles",
              subtitle: "21 Modules Included",
              courseImageLink: 'https://images.unsplash.com/photo-1481627834876-b7833e8f5570?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60',
              rank: 4.8
            },
            {
              course: "Evolution Principle",
              subtitle: "21 Modules Included",
              company: "Hobo Tech",
              courseImageLink: 'https://images.unsplash.com/photo-1521587760476-6c12a4b040da?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60',
              rank: 4.8
            },
            {
              course: "The Game Theories",
              subtitle: "21 Modules Included",
              company: "Hobo Tech",
              courseImageLink: 'https://images.unsplash.com/photo-1491841550275-ad7854e35ca6?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=667&q=80',
              rank: 4.8
            },
            {
              course: "Art & Design Principles",
              subtitle: "21 Modules Included",
              company: "Hobo Tech",
              courseImageLink: 'https://images.unsplash.com/3/doctype-hi-res.jpg?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60',
              rank: 4.8
            }
          ],
        };
      },
    }
  };
</script>

<style scoped>
.all-courses {
  margin-top: 4rem;
}
.all-courses-header {
  color: #2B1C1C;
}
.view-all-course {
  /* View All */

  font-style: italic;
  font-weight: normal;
  font-size: 18px;
  line-height: 23px;
  letter-spacing: 0.05em;
  text-decoration-line: underline;
  text-align: right;

  color: #FF2E2E;
}
</style>