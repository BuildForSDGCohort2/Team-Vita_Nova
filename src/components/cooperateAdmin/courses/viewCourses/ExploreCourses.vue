<!--suppress ALL -->
<template>
  <v-responsive>

    <br>
    <Menu/>

    <br>
    <v-card max-width="1450" class="mx-auto overflow-hidden" elevation="0">
      <EmployeeInviteSent/>
      <h3
        style="padding-left:10px;
               padding-bottom: 20px;
               font-family: IBM Plex Sans;
                font-style: normal;
                font-weight: 600;
                font-size: 36px;
    "
      >
        Top Software Development Courses
      </h3>

      <v-container>
        <v-row>
          <v-col v-for="(soft, i) in software" :key="i" cols="6">
            <v-card :color="soft.color" elevation="0">
              <div class="d-flex flex-no-wrap justify-space-between">
                <v-avatar class="ma-3" size="150" tile>
                  <v-img :src="soft.src"></v-img>
                </v-avatar>

                <div>
                  <v-card-subtitle
                    v-text="soft.course"
                    style="color: #FF2E2E; padding-bottom: 0px"
                  ></v-card-subtitle>
                  <v-card-title
                    class="headline"
                    v-text="soft.title"
                  ></v-card-title>

                  <v-card-subtitle v-text="soft.artist"></v-card-subtitle>
                </div>
              </div>
            </v-card>
          </v-col>
        </v-row>
        <v-p @click="showAll" style="color: #FF2E2E; border: 0px"
          >Show All >></v-p
        >
      </v-container>

      <h3
        style="padding-left:10px;
               padding-bottom: 20px;
               padding-top: 8%;
               font-family: IBM Plex Sans;
                font-style: normal;
                font-weight: 600;
                font-size: 36px;
    "
      >
        Top Business Management Courses
      </h3>

      <v-container>
        <v-row>
          <v-col v-for="(busi, i) in business" :key="i" cols="6">
            <v-card :color="busi.color" elevation="0">
              <div class="d-flex flex-no-wrap justify-space-between">
                <v-avatar class="ma-3" size="125" tile>
                  <v-img :src="busi.src"></v-img>
                </v-avatar>

                <div>
                  <v-card-subtitle
                    v-text="busi.course"
                    style="color: #FF2E2E; padding-bottom: 0px"
                  ></v-card-subtitle>
                  <v-card-title
                    class="headline"
                    v-text="busi.title"
                  ></v-card-title>

                  <v-card-subtitle v-text="busi.artist"></v-card-subtitle>
                </div>
              </div>
            </v-card>
          </v-col>
        </v-row>
        <v-p @click="showAll" style="color: #FF2E2E">Show All >></v-p>
      </v-container>

      <h3
        style="padding-left:10px;
               padding-bottom: 20px;
               padding-top: 8%;
               font-family: IBM Plex Sans;
                font-style: normal;
                font-weight: 600;
                font-size: 36px;
    "
      >
        Top Design Thinking Courses
      </h3>

      <v-container>
        <v-row>
          <v-col v-for="(design, i) in designs" :key="i" cols="6">
            <v-card :color="design.color" elevation="0">
              <div class="d-flex flex-no-wrap justify-space-between">
                <v-avatar class="ma-3" size="125" tile>
                  <v-img :src="design.src"></v-img>
                </v-avatar>

                <div>
                  <v-card-subtitle
                    v-text="design.course"
                    style="color: #FF2E2E; padding-bottom: 0px"
                  ></v-card-subtitle>

                  <v-card-title
                    class="headline"
                    v-text="design.title"
                  ></v-card-title>

                  <v-card-subtitle v-text="design.artist"></v-card-subtitle>
                </div>
              </div>
            </v-card>
          </v-col>
        </v-row>
        <v-p @click="showAll" style="color: #FF2E2E">Show All >></v-p>
      </v-container>
    </v-card>
  </v-responsive>
</template>

<script>
import Menu from "@/components/cooperateAdmin/courses/viewCourses/Menu";
import EmployeeInviteSent from "@/components/splashPages/EmployeeInviteSent";
export default {
  components: {EmployeeInviteSent, Menu},
  data: () => ({
    software: [
      {
        color: "#F8F8F8",
        src: "https://cdn.vuetifyjs.com/images/cards/foster.jpg",
        title: "Introduction to Java For Web Development",
        artist: "Wale Adenuga - 6hrs 30minutes",
        course: "Web Development"
      },

      {
        color: "#F8F8F8",
        src: "https://cdn.vuetifyjs.com/images/cards/foster.jpg",
        title: "Introduction to Java For Web Development",
        artist: "Wale Adenuga - 6hrs 30minutes",
        course: "Web Development"
      },
      {
        color: "#F8F8F8",
        src: "https://cdn.vuetifyjs.com/images/cards/foster.jpg",
        title: "Introduction to Java For Web Development",
        artist: "Wale Adenuga - 6hrs 30minutes",
        course: "Web Development"
      },

      {
        color: "#F8F8F8",
        src: "https://cdn.vuetifyjs.com/images/cards/halcyon.png",
        title: "Introduction to Flutter For Web Development",
        artist: "Ellie Goulding - 6hrs 20minutes",
        course: "Web Development"
      }
    ],

    business: [
      {
        color: "#F8F8F8",
        src: "https://cdn.vuetifyjs.com/images/cards/foster.jpg",
        title: "Introduction to Java For Web Development",
        artist: "Wale Adenuga - 6hrs 30minutes",
        course: "Business Management"
      },

      {
        color: "#F8F8F8",
        src: "https://cdn.vuetifyjs.com/images/cards/foster.jpg",
        title: "Introduction to Java For Web Development",
        artist: "Wale Adenuga - 6hrs 30minutes",
        course: "Web Development"
      },
      {
        color: "#F8F8F8",
        src: "https://cdn.vuetifyjs.com/images/cards/foster.jpg",
        title: "Introduction to Java For Web Development",
        artist: "Wale Adenuga - 6hrs 30minutes",
        course: "Web Development"
      },

      {
        color: "#F8F8F8",
        src: "https://cdn.vuetifyjs.com/images/cards/halcyon.png",
        title: "Introduction to Flutter For Web Development",
        artist: "Ellie Goulding - 6hrs 20minutes",
        course: "Web Development"
      }
    ],

    designs: [
      {
        color: "#F8F8F8",
        src: "https://cdn.vuetifyjs.com/images/cards/foster.jpg",
        title: "Introduction to Java For Web Development",
        artist: "Wale Adenuga - 6hrs 30minutes",
        course: "Business Management"
      },

      {
        color: "#F8F8F8",
        src: "https://cdn.vuetifyjs.com/images/cards/foster.jpg",
        title: "Introduction to Java For Web Development",
        artist: "Wale Adenuga - 6hrs 30minutes",
        course: "Web Development"
      },
      {
        color: "#F8F8F8",
        src: "https://cdn.vuetifyjs.com/images/cards/foster.jpg",
        title: "Introduction to Java For Web Development",
        artist: "Wale Adenuga - 6hrs 30minutes",
        course: "Web Development"
      },

      {
        color: "#F8F8F8",
        src: "https://cdn.vuetifyjs.com/images/cards/halcyon.png",
        title: "Introduction to Flutter For Web Development",
        artist: "Ellie Goulding - 6hrs 20minutes",
        course: "Web Development"
      }
    ]
  }),

  computed: {},

  methods: {
    showAll() {}
  }
};
</script>
