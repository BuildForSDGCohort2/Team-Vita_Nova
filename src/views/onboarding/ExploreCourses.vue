<template>
  <BrowseCourses/>

</template>

<script>

import BrowseCourses from "@/components/onboarding/browseCourses/BrowseCourses";
export default {
  components: {BrowseCourses },
  computed: {}
};
</script>