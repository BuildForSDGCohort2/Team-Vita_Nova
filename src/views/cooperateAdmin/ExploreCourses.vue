<template>
  <div>
    <ExploreCourses />
  </div>
</template>

<script>
import ExploreCourses from "../../components/cooperateAdmin/courses/ExploreCourses/ExploreCourses";
export default {
  components: {
    ExploreCourses
  }
};
</script>