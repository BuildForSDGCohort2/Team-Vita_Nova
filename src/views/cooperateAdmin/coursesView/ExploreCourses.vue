<template>
  <div>
    <ExploreCourses />
  </div>
</template>

<script>
import ExploreCourses from "@/components/cooperateAdmin/courses/exploreCourses/ExploreCourses";
export default {
  components: {
    ExploreCourses
  }
};
</script>