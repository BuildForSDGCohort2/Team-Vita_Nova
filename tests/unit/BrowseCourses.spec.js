import { mount} from '@vue/test-utils'
import ExploreCourses from "@/components/cooperateAdmin/courses/viewCourses/ExploreCourses";



describe('ExploreCourses', ()=>{
    const wrapper=mount(ExploreCourses)
    it('test page has title', () =>{

        expect(wrapper.text()).toContain('Top Software Development Courses')
    })

    it("sets the correct default data", () => {
        expect(typeof ExploreCourses.data).toBe("function");
        const defaultData = ExploreCourses.data();
        expect(defaultData.software).toBe("");
    });
    it("show all button clicks", () => {
        wrapper.find("v-btn").trigger("Show All");
    });
    it("renders a vue instance", () => {
        expect(mount(ExploreCourses).isVueInstance()).toBe(true);
    });


})