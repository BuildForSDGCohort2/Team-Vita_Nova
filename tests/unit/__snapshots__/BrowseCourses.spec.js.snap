// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BrowseCourses render page titles 1`] = `
<v-responsive>
  <div style="font-family: IBM Plex Sans;">
    <v-card max-width="1450" elevation="0" class="mx-auto">
      <v-row class=" align-center justify-center ">
        <h3 style="font-style: normal; font-weight: 300; font-size: 48px; line-height: 62px; margin-top: 5%; display: flex; align-items: center; text-align: center; letter-spacing: 0.6px; text-transform: capitalize; color: rgb(43, 28, 28);">
          Exclusive course to equip your <br>
          team for the best
        </h3>
      </v-row>
      <v-text-field append-icon="mdi-magnify" label="Search by Course name" solo-inverted="" style="max-width: 30%; margin: 5% auto 3% auto;"></v-text-field>
      <h4 style="padding-left: 3%; padding-bottom: 20px; font-style: normal; font-weight: 300; font-size: 24px; line-height: 62px; letter-spacing: 0.6px; text-transform: capitalize; color: rgb(43, 28, 28);">
        Business Management &amp; Marketing Courses
      </h4>
      <v-responsive>
        <v-row class="overlay">
          <v-card cols="6" elevation="2" width="374" class="mx-auto my-12 text-wrap">
            <v-img height="250" src="https://cdn.vuetifyjs.com/images/cards/cooking.png"></v-img>
            <v-card-title>INTRODUCTION TO ROBOTICS</v-card-title>
            <div class="mx-4 subtitle-1">Semicolon Africa</div>
            <v-card-text>
              <v-row align="center" class="mx-0">
                <v-rating value="4.5" color="amber" dense="" half-increments="" readonly="" size="14"></v-rating>
                <div class="grey--text ml-4">4.5 (413)</div>
              </v-row>
            </v-card-text>
          </v-card>
          <v-card cols="6" elevation="2" width="374" class="mx-auto my-12 text-wrap">
            <v-img height="250" src="https://cdn.vuetifyjs.com/images/cards/foster.jpg"></v-img>
            <v-card-title>CREATING STUNNING ENGAGEMENTS</v-card-title>
            <div class="mx-4 subtitle-1">Semicolon Africa</div>
            <v-card-text>
              <v-row align="center" class="mx-0">
                <v-rating value="4.5" color="amber" dense="" half-increments="" readonly="" size="14"></v-rating>
                <div class="grey--text ml-4">4.5 (413)</div>
              </v-row>
            </v-card-text>
          </v-card>
          <v-card cols="6" elevation="2" width="374" class="mx-auto my-12 text-wrap">
            <v-img height="250" src="https://cdn.vuetifyjs.com/images/cards/foster.jpg"></v-img>
            <v-card-title>WEB DEVELOPMENT</v-card-title>
            <div class="mx-4 subtitle-1">Semicolon Africa</div>
            <v-card-text>
              <v-row align="center" class="mx-0">
                <v-rating value="4.5" color="amber" dense="" half-increments="" readonly="" size="14"></v-rating>
                <div class="grey--text ml-4">4.5 (413)</div>
              </v-row>
            </v-card-text>
          </v-card>
          <v-card cols="6" elevation="2" width="374" class="mx-auto my-12 text-wrap">
            <v-img height="250" src="https://cdn.vuetifyjs.com/images/cards/cooking.png"></v-img>
            <v-card-title>INTRODUCTION TO ROBOTICS</v-card-title>
            <div class="mx-4 subtitle-1">Semicolon Africa</div>
            <v-card-text>
              <v-row align="center" class="mx-0">
                <v-rating value="4.5" color="amber" dense="" half-increments="" readonly="" size="14"></v-rating>
                <div class="grey--text ml-4">4.5 (413)</div>
              </v-row>
            </v-card-text>
          </v-card>
          <v-card cols="6" elevation="2" width="374" class="mx-auto my-12 text-wrap">
            <v-img height="250" src="https://cdn.vuetifyjs.com/images/cards/halcyon.png"></v-img>
            <v-card-title>GUIDE TO CREATING MOON MARKETING</v-card-title>
            <div class="mx-4 subtitle-1">Semicolon Africa</div>
            <v-card-text>
              <v-row align="center" class="mx-0">
                <v-rating value="4.5" color="amber" dense="" half-increments="" readonly="" size="14"></v-rating>
                <div class="grey--text ml-4">4.5 (413)</div>
              </v-row>
            </v-card-text>
          </v-card>
          <v-card cols="6" elevation="2" width="374" class="mx-auto my-12 text-wrap">
            <v-img height="250" src="https://cdn.vuetifyjs.com/images/cards/cooking.png"></v-img>
            <v-card-title>INTRODUCTION TO ROBOTICS</v-card-title>
            <div class="mx-4 subtitle-1">Semicolon Africa</div>
            <v-card-text>
              <v-row align="center" class="mx-0">
                <v-rating value="4.5" color="amber" dense="" half-increments="" readonly="" size="14"></v-rating>
                <div class="grey--text ml-4">4.5 (413)</div>
              </v-row>
            </v-card-text>
          </v-card>
          &gt;
        </v-row>
      </v-responsive>
      <h4 style="padding-left: 3%; padding-bottom: 20px; font-style: normal; font-weight: 300; font-size: 24px; line-height: 62px; letter-spacing: 0.6px; text-transform: capitalize; color: rgb(43, 28, 28);">
        Top Business Management Courses
      </h4>
      <v-responsive>
        <v-row class="overlay">
          <v-card cols="6" elevation="2" width="374" class="mx-auto my-12 text-wrap">
            <v-img height="250" src="https://cdn.vuetifyjs.com/images/cards/cooking.png"></v-img>
            <v-card-title>INTRODUCTION TO ROBOTICS</v-card-title>
            <div class="mx-4 subtitle-1">Semicolon Africa</div>
            <v-card-text>
              <v-row align="center" class="mx-0">
                <v-rating value="4.5" color="amber" dense="" half-increments="" readonly="" size="14"></v-rating>
                <div class="grey--text ml-4">4.5 (413)</div>
              </v-row>
            </v-card-text>
          </v-card>
          <v-card cols="6" elevation="2" width="374" class="mx-auto my-12 text-wrap">
            <v-img height="250" src="https://cdn.vuetifyjs.com/images/cards/foster.jpg"></v-img>
            <v-card-title>CREATING STUNNING ENGAGEMENTS</v-card-title>
            <div class="mx-4 subtitle-1">Semicolon Africa</div>
            <v-card-text>
              <v-row align="center" class="mx-0">
                <v-rating value="4.5" color="amber" dense="" half-increments="" readonly="" size="14"></v-rating>
                <div class="grey--text ml-4">4.5 (413)</div>
              </v-row>
            </v-card-text>
          </v-card>
          <v-card cols="6" elevation="2" width="374" class="mx-auto my-12 text-wrap">
            <v-img height="250" src="https://cdn.vuetifyjs.com/images/cards/foster.jpg"></v-img>
            <v-card-title>WEB DEVELOPMENT</v-card-title>
            <div class="mx-4 subtitle-1">Semicolon Africa</div>
            <v-card-text>
              <v-row align="center" class="mx-0">
                <v-rating value="4.5" color="amber" dense="" half-increments="" readonly="" size="14"></v-rating>
                <div class="grey--text ml-4">4.5 (413)</div>
              </v-row>
            </v-card-text>
          </v-card>
          <v-card cols="6" elevation="2" width="374" class="mx-auto my-12 text-wrap">
            <v-img height="250" src="https://cdn.vuetifyjs.com/images/cards/cooking.png"></v-img>
            <v-card-title>INTRODUCTION TO ROBOTICS</v-card-title>
            <div class="mx-4 subtitle-1">Semicolon Africa</div>
            <v-card-text>
              <v-row align="center" class="mx-0">
                <v-rating value="4.5" color="amber" dense="" half-increments="" readonly="" size="14"></v-rating>
                <div class="grey--text ml-4">4.5 (413)</div>
              </v-row>
            </v-card-text>
          </v-card>
          <v-card cols="6" elevation="2" width="374" class="mx-auto my-12 text-wrap">
            <v-img height="250" src="https://cdn.vuetifyjs.com/images/cards/halcyon.png"></v-img>
            <v-card-title>GUIDE TO CREATING MOON MARKETING</v-card-title>
            <div class="mx-4 subtitle-1">Semicolon Africa</div>
            <v-card-text>
              <v-row align="center" class="mx-0">
                <v-rating value="4.5" color="amber" dense="" half-increments="" readonly="" size="14"></v-rating>
                <div class="grey--text ml-4">4.5 (413)</div>
              </v-row>
            </v-card-text>
          </v-card>
          <v-card cols="6" elevation="2" width="374" class="mx-auto my-12 text-wrap">
            <v-img height="250" src="https://cdn.vuetifyjs.com/images/cards/cooking.png"></v-img>
            <v-card-title>INTRODUCTION TO ROBOTICS</v-card-title>
            <div class="mx-4 subtitle-1">Semicolon Africa</div>
            <v-card-text>
              <v-row align="center" class="mx-0">
                <v-rating value="4.5" color="amber" dense="" half-increments="" readonly="" size="14"></v-rating>
                <div class="grey--text ml-4">4.5 (413)</div>
              </v-row>
            </v-card-text>
          </v-card>
          &gt;
        </v-row>
      </v-responsive>
      <h4 style="padding-left: 3%; padding-bottom: 20px; font-style: normal; font-weight: 300; font-size: 24px; line-height: 62px; letter-spacing: 0.6px; text-transform: capitalize; color: rgb(43, 28, 28);">
        Top Design Thinking Courses
      </h4>
      <v-responsive>
        <v-row class="overlay">
          <v-card cols="6" elevation="2" width="374" class="mx-auto my-12 text-wrap">
            <v-img height="250" src="https://cdn.vuetifyjs.com/images/cards/cooking.png"></v-img>
            <v-card-title>INTRODUCTION TO ROBOTICS</v-card-title>
            <div class="mx-4 subtitle-1">Semicolon Africa</div>
            <v-card-text>
              <v-row align="center" class="mx-0">
                <v-rating value="4.5" color="amber" dense="" half-increments="" readonly="" size="14"></v-rating>
                <div class="grey--text ml-4">4.5 (413)</div>
              </v-row>
            </v-card-text>
          </v-card>
          <v-card cols="6" elevation="2" width="374" class="mx-auto my-12 text-wrap">
            <v-img height="250" src="https://cdn.vuetifyjs.com/images/cards/foster.jpg"></v-img>
            <v-card-title>CREATING STUNNING ENGAGEMENTS</v-card-title>
            <div class="mx-4 subtitle-1">Semicolon Africa</div>
            <v-card-text>
              <v-row align="center" class="mx-0">
                <v-rating value="4.5" color="amber" dense="" half-increments="" readonly="" size="14"></v-rating>
                <div class="grey--text ml-4">4.5 (413)</div>
              </v-row>
            </v-card-text>
          </v-card>
          <v-card cols="6" elevation="2" width="374" class="mx-auto my-12 text-wrap">
            <v-img height="250" src="https://cdn.vuetifyjs.com/images/cards/foster.jpg"></v-img>
            <v-card-title>WEB DEVELOPMENT</v-card-title>
            <div class="mx-4 subtitle-1">Semicolon Africa</div>
            <v-card-text>
              <v-row align="center" class="mx-0">
                <v-rating value="4.5" color="amber" dense="" half-increments="" readonly="" size="14"></v-rating>
                <div class="grey--text ml-4">4.5 (413)</div>
              </v-row>
            </v-card-text>
          </v-card>
          <v-card cols="6" elevation="2" width="374" class="mx-auto my-12 text-wrap">
            <v-img height="250" src="https://cdn.vuetifyjs.com/images/cards/cooking.png"></v-img>
            <v-card-title>INTRODUCTION TO ROBOTICS</v-card-title>
            <div class="mx-4 subtitle-1">Semicolon Africa</div>
            <v-card-text>
              <v-row align="center" class="mx-0">
                <v-rating value="4.5" color="amber" dense="" half-increments="" readonly="" size="14"></v-rating>
                <div class="grey--text ml-4">4.5 (413)</div>
              </v-row>
            </v-card-text>
          </v-card>
          <v-card cols="6" elevation="2" width="374" class="mx-auto my-12 text-wrap">
            <v-img height="250" src="https://cdn.vuetifyjs.com/images/cards/halcyon.png"></v-img>
            <v-card-title>GUIDE TO CREATING MOON MARKETING</v-card-title>
            <div class="mx-4 subtitle-1">Semicolon Africa</div>
            <v-card-text>
              <v-row align="center" class="mx-0">
                <v-rating value="4.5" color="amber" dense="" half-increments="" readonly="" size="14"></v-rating>
                <div class="grey--text ml-4">4.5 (413)</div>
              </v-row>
            </v-card-text>
          </v-card>
          <v-card cols="6" elevation="2" width="374" class="mx-auto my-12 text-wrap">
            <v-img height="250" src="https://cdn.vuetifyjs.com/images/cards/cooking.png"></v-img>
            <v-card-title>INTRODUCTION TO ROBOTICS</v-card-title>
            <div class="mx-4 subtitle-1">Semicolon Africa</div>
            <v-card-text>
              <v-row align="center" class="mx-0">
                <v-rating value="4.5" color="amber" dense="" half-increments="" readonly="" size="14"></v-rating>
                <div class="grey--text ml-4">4.5 (413)</div>
              </v-row>
            </v-card-text>
          </v-card>
          &gt;
        </v-row>
      </v-responsive>
    </v-card>
  </div>
</v-responsive>
`;
